runtime/doc/usr_41.txt

```diff
@@ -1636,14 +1636,23 @@ manual: |exception-handling|.
 Here is a summary of items that apply to Vim scripts.  They are also mentioned
 elsewhere, but form a nice checklist.
 
-The end-of-line character depends on the system.  For Unix a single <NL>
-character is used.  For Windows <CR><NL> is used.  This is important when
-using mappings that end in a <CR>.  See |:source_crnl|.
+The end-of-line character depends on the system.  For Vim scripts it is
+recommended to always use the Unix fileformat.  Lines are then separated with
+the Newline character.  This also works on any other system.  That way you can
+copy your Vim scripts from MS-Windows to Unix and they still work.  See
+|:source_crnl|.  To be sure it is set right, do this before writing the file:
+ >
+	:setlocal fileformat=unix
+When using "dos" fileformat, lines are separated with CR-NL, two characters.
+The CR character causes various problems, better avoid this.
 
 
 WHITE SPACE
 
-Blank lines are allowed and ignored.
+Blank lines are allowed in a script and ignored.
+
+Leading whitespace characters (blanks and TABs) are ignored, except when using
+|:let-heredoc| without "trim".
```

Comment:
This is repeated a bit in the paragraph below. Looks like a lot of the below paragraph was removed upstream in https://github.com/vim/vim/commit/04fb916684829f6aa12f33f14d0d0023b458f200

Reply:
Ah, I meant to consolidate that. Thanks!